docs/eosio-home/getting-the-software.rst

```diff
@@ -0,0 +1,138 @@
+1.3 노드 시작 및 설정
+==========================
+
+아직 Docker를 설치하지 않은 상태라면, 다음의 링크에서 다운로드 받을 수 있다: https://www.docker.com/community-edition
+
+.. rubric:: 1단계: Docker 이미지 받기
+
+다음 커맨드를 실행하면 컴파일된 EOSIO 소프트웨어를 포함한 Ubuntu 이미지를 받을 수 있다.
+
+.. code-block:: console
+
+   docker pull eosio/eos:v1.4.3
+
+.. tip:: 2018년 11월 12일 현재 최신 버전은 1.4.3이다.
+
+.. rubric:: 2단계: 노드와 지갑 시작하기
+
+``contracts`` 디렉토리를 생성한 단계의 마지막에서 절대 경로명을 파악했을 것이다. 이 ``contracts`` 디렉토리에 대한 절대 경로로 아래 커맨드의 "CONTRACTS_DIR" 부분을 모두 대체한다.
+
+.. code-block:: console
+
+  docker run --name eosio \
+    --publish 7777:7777 \
+    --publish 127.0.0.1:5555:5555 \
+    --volume CONTRACTS_DIR:CONTRACTS_DIR \
+    --detach \
+    eosio/eos:v1.4.3 \
+    /bin/bash -c \
+    "keosd --http-server-address=0.0.0.0:5555 & exec nodeos -e -p eosio --plugin eosio::producer_plugin --plugin eosio::chain_api_plugin --plugin eosio::history_plugin --plugin eosio::history_api_plugin --plugin eosio::http_plugin -d /mnt/dev/data --config-dir /mnt/dev/config --http-server-address=0.0.0.0:7777 --access-control-allow-origin=* --contracts-console --http-validate-host=false --filter-on='*'"
+
+이 커맨드를 통해 다음과 같은 설정이 이루어진다.
+
+#. 7777번과 5555번 포트를 호스트 시스템과 연동(forward)한다.
+#. 로컬 드라이브에 있는 작업 디렉토리를 컨테이너에 연결한다.
+#. bash를 통해 nodeos를 시작한다. 여기서는 모든 기본 플러그인들을 로드하고 서버 주소를 설정하며, CORS와 컨트랙트 디버그 기능을 활성화한다.
+#. CORS를 아무런 제약이 없는 상태로(*) 설정한다.
+
+.. warning:: 위의 설정에서 CORS를 모두( `*` )에게 허용한 것은 **개발 과정에서 사용하기 위한 용도로만** 그렇게 한 것이며, 외부로 공개된 노드의 CORS를 **절대로** 모두( `*` )에게 허용해서는 안된다!
```

Comment:
```suggestion
.. warning:: 위의 설정에서 CORS를 모두( ``*`` )에게 허용한 것은 **개발 과정에서 사용하기 위한 용도로만** 그렇게 한 것이며, 외부로 공개된 노드의 CORS를 **절대로** 모두( ``*`` )에게 허용해서는 안된다!
```

Reply:
RST는 Markdown과 달리 inline literal에 대해 back quote(`)를 두 개씩 사용해야 합니다

Reply:
ㅠㅠ 메일이 스팸함으로 들어와 있는 바람이 방금 봤네요, back quote 하나는 reference (link)인거죠?

Reply:
rst에서는 용도가 명확하지 않은 것 같습니다. [link](http://docutils.sourceforge.net/docs/user/rst/quickref.html)의 Inline Markup 항목에 보시면 아래와 같이 설명되어 있네요.
```
The rendering and meaning of interpreted text is domain- or application-dependent. It can be used for things like index entries or explicit descriptive markup (like program identifiers). 
```